contrib/ansible/roles/skydive_analyzer/tasks/docker.yml

```diff
@@ -10,6 +10,34 @@
     state: started
     enabled: yes
 
+- name: Run docker registry
+  shell: "docker run -d -p 5000:5000 --restart=always --name registry registry:2"
+  when: '"file://" in skydive_binary_remote_location'
```

Comment:
not sure you need to surround the while line by with '

Reply:
it seems so